src/compiler/parser.cc

```diff
@@ -83,7 +83,7 @@ namespace verona::compiler
 
     Rule keyword = term(
       "while"_E | "if" | "class" | "interface" | "primitive" | "var" | "unit" |
-      "match" | "String" | "iso" | "mut" | "imm" | "mut-view" | "in" | "cown" |
+      "match" | "String" | "iso" | "mut" | "imm" | "mut-view" | "in" |
```

Comment:
Can user program now define a cown type? I guess core library will define, an if we get two, then we error?

Reply:
Yes and yes. If and when we get modules, it will be possible to define a type cown in a different namespace.